cmd/benchmark/main.go

```diff
@@ -226,18 +255,25 @@ func getTotalBalance(insSDK *sdk.SDK, members []*sdk.Member) (totalBalance *big.
 	return totalBalance
 }
 
-func getMembers(insSDK *sdk.SDK) ([]*sdk.Member, error) {
-	var members []*sdk.Member
+func getMembers(insSDK *sdk.SDK, number int, migration bool) ([]sdk.Member, error) {
+	var members = make([]sdk.Member, number)
 	var err error
 
 	if useMembersFromFile {
-		members, err = loadMembers(concurrent * 2)
+		for i := range members {
```

Comment:
Why don't create `make([]*sdk.MigrationMember, number)` or `make([]*sdk.CommonMemberr, number)`, and don't do this?

Reply:
We will need different loadMembers method then.